src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerListImages.groovy

```diff
@@ -16,13 +16,12 @@
 package com.bmuschko.gradle.docker.tasks.image
 
 import com.bmuschko.gradle.docker.response.ResponseHandler
-import com.bmuschko.gradle.docker.response.image.ListImagesResponseHandler
 import com.bmuschko.gradle.docker.tasks.AbstractDockerRemoteApiTask
 import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.Optional
 
 class DockerListImages extends AbstractDockerRemoteApiTask {
-    private ResponseHandler<Void, List<Object>> responseHandler = new ListImagesResponseHandler()
+    private ResponseHandler<Void, List<Object>> responseHandler
```

Comment:
Because there is a setter, I plan to leave it as is to be backward compatible in case someone provided custom handler. I will add deprecation and we can remove it in future release.